<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIBody</class>
 <widget class="QWidget" name="UIBody">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>1175</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="16" column="0">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Gravity Scale</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cbASleep">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLineEdit" name="leADemping"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Allow Sleep</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLineEdit" name="leLDemping"/>
      </item>
      <item row="20" column="0" colspan="2">
       <widget class="QComboBox" name="cbType">
        <item>
         <property name="text">
          <string>Static</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dynamic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kinematic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="leAVelocity"/>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="leLVelocity_x"/>
        </item>
        <item>
         <widget class="QLineEdit" name="leLVelocity_y"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Fixed Rotation</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Bullet</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QComboBox" name="cbActive">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Angular Velocity</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0" colspan="2">
       <widget class="QLineEdit" name="leGScale"/>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="cbAwake">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Angular Damping</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Linear Damping</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Awake</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="cbBullet">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="cbFRotation">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Linear Velocity</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Auto scaling</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QComboBox" name="cbEnableScale">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ggbShape">
     <layout class="QGridLayout" name="lyShapeEditor">
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Friction</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Sensor</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Restitution</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Density</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cbSensor">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="leRestitution"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="leFriction"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="leDensity"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ggbVertex">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Vertex</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="leVertex_x"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leVertex_y"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listShapes">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbAddShape">
     <property name="text">
      <string>Add Shape</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbDisableShape">
     <property name="text">
      <string>Disable Shape</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbDeleteShape">
     <property name="text">
      <string>Delete Shape</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
